### ISSUE TITLE
[Code Structure] Follow protobuf enum best practices

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/pocket#258]
## Objective

Not following best practices for protobuf enum naming.

## Origin Document

In #245, the following change was made:

<img width="1228" alt="Screen Shot 2022-09-29 at 12 06 46 PM" src="https://user-images.githubusercontent.com/1892194/193120710-ce1e72ce-a522-420b-a9cc-85dd588a1df9.png">

In order to follow [best practices](https://developers.google.com/protocol-buffers/docs/style), we should do the following:

<img width="886" alt="Screen Shot 2022-09-29 at 12 08 20 PM" src="https://user-images.githubusercontent.com/1892194/193124479-86a0d829-9cb6-481b-b74e-f337f12c7ea4.png">

## Goals

- Change all existing protobuf enums to follow best practices
- Enforce best practices moving forward

## Deliverable

- [ ] A single PR that converts existing protobufs (and their references) to follow best practices
- [ ] A single PR to update to update workflow to enforce best practices

## Non-goals / Non-deliverables

- Changing any business logic in the codebase

## General issue deliverables

- [ ] Update the appropriate CHANGELOG
- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references

## Testing Methodology

- **All tests**: `make test_all`
- **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)

---

**Creator**: @Olshansk 
**Co-Owners**: @okdas + ?

**Comment by nickatnight23** [comment]
Can I take this on?

**Comment by Olshansk** [comment]
@nickatnight23 Definitely! Feel free to reach out either here or in discord if you need any help or clarifications.

The (dev guide](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) is probably the best place to start, but if something is unclear or could be made better, please don't hesitate to point it out

**Comment by nickatnight23** [comment]
Can I get some clarification on installing the dependencies? I am having trouble with this section- protoc-gen-go, protoc-go-inject-tag and mockgen by running make install_cli_deps. Not sure how to make the build first?

**Comment by Olshansk** [comment]
Can you try running this command: https://github.com/pokt-network/pocket/blob/main/Makefile#L99

If it works, feel free to update the dev guide as well!

**Comment by nickatnight23** [comment]
I have a question about the code base where the protobuf enums are located. Would this be the best place to ask?

**Comment by Olshansk** [comment]
For sure. Are you just having trouble finding them? This command might be a good starting point:

```bash
$ find . -name "*.proto" | xargs grep -i "enum "
```

**Comment by nickatnight23** [comment]
Yes, I was having trouble finding them. Thank you.

**Comment by nickatnight23** [comment]
I am having trouble installing mockgen. It appears that everything else is installed.

**Comment by Olshansk** [comment]
What is the issue?
